library/private/animationManager.h

```diff
@@ -103,27 +99,31 @@ class animationManager
    */
   bool LoadAtTime(double timeValue);
 
-  animationManager(animationManager const&) = delete;
```

Comment:
Clang-tidy is suppose to warning about this.  
Read https://discourse.llvm.org/t/why-does-clang-tidy-recommend-deleted-member-functions-should-be-public/46728

Reply:
I've fixed it but no idea no warnings